src/omnibase_core/errors/error_code_utils.py

```
@@ -333,35 +220,13 @@ def list_registered_components() -> list[str]:
-
-# ModelRegistryErrorModel has been extracted to model_registry_error_model.py
+__all__ = [
+    "STATUS_TO_EXIT_CODE",
+    "CORE_ERROR_CODE_TO_EXIT_CODE",
+    "get_exit_code_for_status",
+    "get_exit_code_for_core_error",
+    "get_core_error_description",
+    "register_error_codes",
+    "get_error_codes_for_component",
+    "list_registered_components",
+]
```

Ruff (0.14.2)

`__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)